README.md

```diff
@@ -1,3 +1,34 @@
-# Extension Activity Monitor
+# Extension Activity Monitor [![Build Status](https://travis-ci.org/mozilla/extension-activity-monitor.svg?branch=master)](https://travis-ci.org/mozilla/extension-activity-monitor) [![Coverage Status](https://coveralls.io/repos/github/mozilla/extension-activity-monitor/badge.svg?branch=master)](https://coveralls.io/github/mozilla/extension-activity-monitor?branch=master)
 
-TBD
+This is a privileged extension for Firefox that monitors the activities of other installed extensions. The extension uses a privileged API called `activityLog` API to retrieve logs from other installed extensions.
+
+## Installation
+
+### Prerequisite
+
+- Clone / download the respository in your local machine.
+- Install [Firefox Nightly](https://www.mozilla.org/en-US/firefox/all/#product-desktop-nightly) to run this privileged extension.
+
+### Using [web-ext](https://github.com/mozilla/web-ext)
+
+- Run `npm run start:firefox` to load the extension temporarily in the Firefox Nightly.
+
+_NOTE_: You need to open Firefox Nightly previously to run the above command. If Firefox Nightly is undetectable or if you have installed it in a different location you may try the following command:
```

Comment:
Nit (just a couple of tweaks, and a shorter command to override just the firefox binary, if we do include the rest of the options in the web-ext configuration as suggested in my previous comment): 

_NOTE_: You may need to run Firefox Nightly once to let the above command to auto-discovery where the Firefox Nightly binary is located in your system.
Alternatively the Firefox binary location can be provided manually on the command line:
```
$ web-ext run -f /path/to/firefox-nightly/firefox
```

Reply:
Thanks for the suggestion. Added it.